eis_qgis_plugin/processing/algorithms/filtering/gaussian_filter.py

```diff
@@ -0,0 +1,78 @@
+from qgis.core import (
+    QgsProcessingParameterNumber,
+    QgsProcessingParameterRasterDestination,
+    QgsProcessingParameterRasterLayer,
+)
+
+from eis_qgis_plugin.processing.eis_processing_algorithm import EISProcessingAlgorithm
+
+
+class EISGaussianFilter(EISProcessingAlgorithm):
+    def __init__(self) -> None:
+        super().__init__()
+
+        self._name = "gaussian_filter"
+        self._display_name = "Gaussian filter"
+        self._group = "Filtering"
+        self._group_id = "filtering"
+        self._short_help_string = "Apply a basic gaussian filter to the input raster"
+
+    def initAlgorithm(self, config=None):
+        self.alg_parameters = [
+            "input_raster",
+            "sigma",
+            "truncate",
+            "size",
+            "output_raster",
+        ]
+
+        input_raster_param = QgsProcessingParameterRasterLayer(
+                name=self.alg_parameters[0], description="Input raster"
+            )
+        input_raster_param.setHelp("The input raster data set.")
+        self.addParameter(input_raster_param)
+
+        sigma_param = QgsProcessingParameterNumber(
+            name=self.alg_parameters[1],
+            description="Sigma",
+            minValue=1.0,
+            defaultValue=1.0,
+            type=QgsProcessingParameterNumber.Double,
```

Comment:
Not 100% sure but I think sigma just needs to be > 0, so again maybe minValue to something like 0.001